### DISCUSSION TITLE
Cant get the Desktop Install to start GUI

### DISCUSSION THREAD

**Discussion by DonMVB** [Security-Onion-Solutions/securityonion#13997] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

88

### Storage for /nsm

169

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

My objective is to have a SO install to support a Malware Lab for my SOC team. To that end I setup Sec Onion in "stand alone", and the GUI works properly when you point a desktop web browser to the DNS name and/or the IP address. I would like to have two "SO desktop" instances available so that I don't need a mini-fleet of Windows PC's just to get to the desktop. To that end, I installed the "desktop" option from the ISO, but I cannot figure out how to get a graphical desktop started on the new machine. 

I did connect and authorize the "desktop" to the "standalone", and the various indicators in the Grid section show green. 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
The desktop GUI components are configured during the first highstate -- log into the new Desktop node and run "sudo so-checkin", confirm that it completes without errors, and then reboot.

**Comment by nightstalker117** [comment]
running "sudo so-checkin" from the desktop node.  I get the following error message: 

     "sudo: so-checkin: command not found"

**Comment by InfosecGoon** [comment]
And you said the new Desktop node shows up in Grid in the Manager?
